app/src/main/res/values/strings.xml

```diff
@@ -444,6 +447,15 @@
     <string name="searchable_settings_description">Media</string>
     <string name="pref_subtitles_background_title">Enable background</string>
     <string name="pref_subtitles_background_summary">Show a black background behind the subtitles</string>
+    <string name="pref_subtitles_color_white">White</string>
+    <string name="pref_subtitles_color_black">Black</string>
+    <string name="pref_subtitles_color_darkgrey">Dark Grey</string>
+    <string name="pref_subtitles_color_red">Red</string>
+    <string name="pref_subtitles_color_green">Green</string>
+    <string name="pref_subtitles_color_blue">Blue</string>
+    <string name="pref_subtitles_color_yellow">Yellow</string>
+    <string name="pref_subtitles_color_pink">Pink</string>
+    <string name="pref_subtitles_color_cyan">Cyan</string>
```

Comment:
The strings can probably just be `color_white` etc. so we can reuse them if we ever need the color names in another place.